datumaro/plugins/open_images_format.py

```diff
@@ -78,6 +86,30 @@ class OpenImagesPath:
         'Confidence',
     )
 
+    BBOX_DESCRIPTION_FIELDS = (
+        'ImageID',
+        'Source',
+        'LabelName',
+        'Confidence',
+        'XMin',
+        'XMax',
+        'YMin',
+        'YMax',
+        'IsOccluded',
+        'IsTruncated',
+        'IsGroupOf',
+        'IsDepiction',
+        'IsInside',
+    )
+
+    BBOX_BOOLEAN_ATTRIBUTES = (
+        types.SimpleNamespace(datumaro_name='occluded', oid_name='IsOccluded'),
+        types.SimpleNamespace(datumaro_name='truncated', oid_name='IsTruncated'),
+        types.SimpleNamespace(datumaro_name='is_group_of', oid_name='IsGroupOf'),
+        types.SimpleNamespace(datumaro_name='is_depiction', oid_name='IsDepiction'),
+        types.SimpleNamespace(datumaro_name='is_inside', oid_name='IsInside'),
```

Comment:
Having `is_` only in some boolean attributes looks a bit misaligned. But let's keep it.

Reply:
Yeah, I'm not a big fan of it either. The main reason I kept the "is" is that "group" and "depiction" are nouns rather than adjectives (and "inside" can be a noun as well), so it would be a bit unnatural to have them as boolean attributes.